Objects/regions.c

```diff
@@ -92,9 +114,265 @@ struct regionmetadata {
     regionmetadata* next;
 };
 
-static bool is_bridge_object(PyObject *op);
-static int regionmetadata_has_ancestor(regionmetadata* data, regionmetadata* other);
-static regionmetadata* PyRegion_get_metadata(PyRegionObject* obj);
+static Py_region_ptr_t regionmetadata_get_representative(Py_region_ptr_t self)
+{
+    // Test for constants
+    if (self == 0 || IS_CONST_REGION(self)) {
```

Comment:
Is the left disjunct the local region here?  Please don't use `0`
```suggestion
    if (self == 0 || IS_CONST_REGION(self)) {
```

Reply:
Nope, the default region now has the value `1<<3` This is intended to handle null pointers, but since the type is `Py_region_ptr_t` it triggers a warning with `== NULL`

Reply:
Perhaps, have a macro for null check, so it is clearer to read.